site/source/user-manual/backups/backup-setup/backup-physical.rst

```diff
@@ -0,0 +1,38 @@
+.. _backup-physical:
+
+=================================
+Backup to Secondary Embassy Drive
```

Comment:
Secondary Embassy Drive doesn't sounds quite right. Physical Drive is better, even though it collides with the fact that you can use LAN for physical drive as well. Maybe there is a 3rd option.

Reply:
Going with "Powered External Drive" for now - open to changes